app/server/lib/ActionHistoryImpl.ts

```diff
@@ -17,9 +17,9 @@ import {ISQLiteDB, ResultRow} from './SQLiteDB';
 // on rows and the maximum total number of bytes in the "body" column.
 // Pruning is done when the history has grown above these limits, to
 // the specified factor.
-const ACTION_HISTORY_MAX_ROWS = 1000;
-const ACTION_HISTORY_MAX_BYTES = 1000 * 1000 * 1000;  // 1 GB.
-const ACTION_HISTORY_GRACE_FACTOR = 1.25;  // allow growth to 1250 rows / 1.25 GB.
+const ACTION_HISTORY_MAX_ROWS = process.env.ACTION_HISTORY_MAX_ROWS || 1000;
+const ACTION_HISTORY_MAX_BYTES = process.env.ACTION_HISTORY_MAX_BYTES || 1000 * 1000 * 1000;  // 1 GB.
```

Comment:
You may rather consider using the `section` util, like this:
```
const ACTION_HISTORY_MAX_BYTES = section.flag('actionHistoryMaxBytes').requireInt({
      envVar: 'ACTION_HISTORY_MAX_BYTES',
      defaultValue: 1000 * 1000 * 1000,
    });
```

This way, you are sure about the type, and you have default value.
Also if the passed string is not a number, it raises an error for you and the server crashes immediately (instead of having erratic behavior).

You may take a look at [OIDCConfig](https://github.com/gristlabs/grist-core/blob/main/app/server/lib/OIDCConfig.ts) for an example.

Reply:
Ah, I used `parseInt` instead, does that still work ?

Reply:
> Ah, I used parseInt instead, does that still work ?

To work, you should check that the result of that function does not return `NaN` and throw otherwise. 

Also it feels more future-proof (in case Grist stores the configuration in a file or in the database).

> Also, I wouldn't know in which "section" to put it ? Should I create a new one ?

I would say so.

Also I would control the minimum value (and maybe the maximum). There is no such thing in the current implementation of `readInt` / `requireInt`, I would suggest to do that. If you need help on it, or need me to make this addition, please ask.